src/main/resources/static/js/popup/validation.js

```diff
@@ -0,0 +1,136 @@
+function RegisterValidation(data) {
+  const formEl = document.getElementById(data.id);
+  const formField = formEl.elements;
+
+  let errors = [];
+  const rulesPattern = {
+    "email-reg": /^\w{1,}@\w{1,}\.\w{2,}$/,
+  };
+
+  const validatorMethods = {
+    notEmpty(el) {
+      return el.value !== "";
+    },
+    pattern(el, pattern) {
+      return rulesPattern[pattern].test(el.value);
+    },
+    contains(el, el2) {
+      if (el.value === formField[el2].value) {
+        formField[el2].nextElementSibling.innerHTML = "";
+        return true;
+      } else return false;
+    },
+    notMore(el) {
+      return el.value.length < 15;
```

Comment:
Why did you hard-coded 15 here? Please move it to constant or variable with name that will describe it's purpose.